nuxt.config.ts

```diff
@@ -99,13 +99,13 @@ export default defineNuxtConfig({
       apiToken: '',
     },
     public: {
-      privacyPolicyUrl: 'https://mozilla.social/privacy-policy',
+      privacyPolicyUrl: process.env.NUXT_PUBLIC_PRIVACY_POLICY_URL || '',
```

Comment:
@bassrock Do we need to call these out explicitly? I think Nuxt automatically pulls env variables with the appropriate prefix: https://nuxt.com/docs/guide/going-further/runtime-config#environment-variables

Reply:
Hmm @aaga we should give it a try, probably not!